modules/ardrivo/src/Servo.cxx

```diff
@@ -20,28 +20,27 @@
 #include "BoardDataDef.hxx"
 #include "Servo.h"
 
-#define SERVO_MIN() (MIN_PULSE_WIDTH - this->min * 4)
-#define SERVO_MAX() (MAX_PULSE_WIDTH - this->max * 4)
+constexpr int SERVO_MIN(int8_t min) {return MinPulseWidth - min * 4;}
+constexpr int SERVO_MAX(int8_t max) {return MaxPulseWidth - max * 4;}
```

Comment:
SHOUTING_SNAKE_CASE is reserved for macros
Also, your functions use identifiers that are not in scope

Reply:
What did you mean by static linkage?

Reply:
`static`
But you made them `constexpr`, which is better, even though really these functions do not seem to have much of a purpose since they're only used once

Reply:
So I will leave it like this then

Reply:
You still need to fix the casing; functions identifiers should be snake_case

Reply:
Fixed already